.eslintrc

```diff
@@ -10,6 +10,7 @@
    "plugins": [
    ],
    "rules": {
-    "no-console":0
+    "no-console": 0,
+    semi: [2, "always"]
```

Comment:
I've now changed it so that eslint will warn about forgotten semicolons (previously it would error and exit with a non-zero exit code). Would you be ok with this compromise?